example/gardener-local/controlplane/values.yaml

```diff
@@ -166,6 +166,49 @@ global:
     workloadIdentity:
       token:
         issuer: https://issuer.gardener.cloud.local
+        signingKey:
+          tls:
+            key: |
+              -----BEGIN PRIVATE KEY-----
```

Comment:
Can we generate this during deployment so it is not hard coded? I am afraid that someone can use this to establish trust from a cloud provider for testing purposes 🤔

Reply:
I get your concerns, will check if this is possible.